### ISSUE TITLE
gridpack generation problem while using CMSConnect

### ISSUE THREAD

**Issue by gourangakole** [cms-sw/genproductions#1420]
Hello Experts
card I am using: 
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll

I see there is an error
++ tee /srv/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll.log
gridpack_generation.sh: line 105: /dev/fd/62: No such file or directory
++ tee /srv/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll.log

genproduction master upto PR #1405

I have used command like below:
nohup ./submit_cmsconnect_gridpack_generation.sh  <name of process card without _proc_card.dat> <folder containing cards relative to current location> > somefile.debug 2>&1 &

**Comment by khurtado** [comment]
I think this is because singularity has issues with process substitution >()
Just replaced it with an alternative and made a PR in #1421, but I haven't tested it.

@gourangakole : Could you test this PR?

**Comment by gourangakole** [comment]
@khurtado thanks a lot. I am running with the updated gridpack_generation.sh script for last couple of hours.
Till now I see its running fine.
Once my gridpack is done I shall update.

**Comment by gourangakole** [comment]
To update this issue:
I see my gridpack generation is finished by at the end I see a gridpack but there is some error while running madspin:

INFO: Restrict model /stash2/user/gkole/genproductions/bin/MadGraph5_aMCatNLO/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_gri\
dpack/work/MG5_aMC_v2_4_2/models/loop_sm with file ../MG5_aMC_v2_4_2/models/loop_sm/restrict_ckm_no_b_mass.dat .
INFO: Run "set stdout_level DEBUG" before import for more information.
^[[1;31mError when reading /stash2/user/gkole/genproductions/bin/MadGraph5_aMCatNLO/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToA\
ll_gridpack/work/processtmp/tmp9AH5Vd/temp_decay/SubProcesses/P0_wp_cdx/G1/results.dat^[[0m
^[[1;31mCommand "launch -n pilotrun" interrupted with error:
ValueError : need more than 5 values to unpack
Please report this bug on https://bugs.launchpad.net/mg5amcnlo
More information is found in '/stash2/user/gkole/genproductions/bin/MadGraph5_aMCatNLO/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_V\
ToAll_gridpack/work/processtmp/pilotrun_tag_1_debug.log'.
Please attach this file to your report.^[[0m

Please find the full log here:
https://gkole.web.cern.ch/gkole/public/MC_Contact/Madgraph5_aMCatNLO/CMSConnect/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll.log

**Comment by khurtado** [comment]
That doesn't seem like a CMS connect issue anymore. You might want to create another thread  (either here or on hypernews: https://hypernews.cern.ch/HyperNews/CMS/get/generators.html?, for it to have more visibility)

**Comment by gourangakole** [comment]
Thanks @khurtado  
I have sent a mail there:
https://hypernews.cern.ch/HyperNews/CMS/get/generators/3651.html

**Comment by perrozzi** [comment]
can be https://github.com/cms-sw/genproductions/pull/1421 merged then?

**Comment by khurtado** [comment]
I would say yes.

**Comment by kdlong** [comment]
I suggest that this be closed. @gourangakole can you open a new issue if the original problem wasn't solved? It's best for us to keep the scope of issues as narrow as possible.

**Comment by gourangakole** [comment]
@kdlong  this is solved in other way. But I would close this since #1421 is also helped.
